### DISCUSSION TITLE
BBT sentence case disapper

### DISCUSSION THREAD

**Discussion by luxaug** [retorquere/zotero-better-bibtex#2595] [Q&A] [answered]
"BBT sentence case" has disappeared as an option in the title casing context menu after I updated BBT to v6.7.94. Is this a bug? Is the option dropped?

**Comment by retorquere** [comment, marked as answer]
I'm trying to bring it back, but a lot has changed in the run-up to the Zotero 7 release.

**Reply by luxaug** [reply]
Thanks 😀
